cnf-tests/testsuites/e2esuite/test_suite_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	_ "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/sctp"                       // this is needed otherwise the sctp test won't be executed
 	_ "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/security"                   // this is needed otherwise the security test won't be executed
 	_ "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/sro"                        // this is needed otherwise the sro test won't be executed
+	_ "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/vrf"                        // this is needed otherwise the performance test won't be executed
```

Comment:
Wasn't it removed in the previous commit? If so, can we adjust it there?
Also, the comment is wrong

Reply:
the import of the package was removed since we don't need it. what i added here is for execution, what we previously didn't have.

Reply:
But this is fixing something that was broken in the previous commit because the import was removed?

Reply:
we onlt had an import there to get the ns, which we don't need anymore. we never had the import for test execution.

Reply:
Still, that commit broke the execution of vrf tests

Reply:
they ran here successfully before i even added the second commit with the execution import https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-kni_cnf-features-deploy/1195/pull-ci-openshift-kni-cnf-features-deploy-master-e2e-gcp-ovn/1553643919286210560

Reply:
yup, but they didn't run the vrf tests I guess. That's what I meant by breaking it.

Reply:
i meant the vrf tests were running in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-kni_cnf-features-deploy/1195/pull-ci-openshift-kni-cnf-features-deploy-master-e2e-gcp-ovn/1553643919286210560